test/new-e2e/runner/ci_profile.go

```diff
@@ -15,7 +15,7 @@ import (
 
 const defaultCISecretPrefix = "ci.datadog-agent."
 
-var defaultCIEnvs = []string{"aws/agent-qa"}
+var defaultCIEnvs = "aws/agent-qa"
```

Comment:
s/var/const to be bundled with above `const`, name to be aligned with the one in `local_profile.go`.